src/components/addIcon/AddIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { IoMdPersonAdd } from 'react-icons/io';
+import { IconContext } from 'react-icons';
+import 'src/components/addIcon/addIcon.scss';
```

Comment:
```suggestion
import './addIcon.scss';
```
Usa la ruta así (como todo el mundo <3)